### PR TITLE
Clean up MP Telemetry 1.0

### LINKED ISSUE #22281: Make mpTelemetry-1.0 not tolerate EE9

### PR DESCRIPTION
**Pull request by tevans78**
Remove com.fasterxml.jackson.jr
Don't tolerate EE9 or MP5
Require Java 11
Remove unused imports
Tidy formatting

#build
fixes https://github.com/OpenLiberty/open-liberty/issues/22281

### CODE REVIEW

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10/server.xml [hunk 1]
```diff
@@ -3,11 +3,8 @@
 	<include location="../fatTestPorts.xml" />
 
 	<featureManager>
-		<feature>cdi-3.0</feature>
-		<feature>mpConfig-3.0</feature>
-		<feature>servlet-5.0</feature>
-        <feature>mpRestClient-3.0</feature>
-		<feature>mpTelemetry-1.0</feature>
+		<feature>servlet-6.0</feature>
+                <feature>mpTelemetry-1.0</feature>
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10/server.xml:7]: Is `mpRestClient-3.0` not needed for this FAT? It's not included as a dependency of `mpTelemetry-1.0`.

File: dev/io.openliberty.org.testcontainers/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+cache
```
Comment (Azquelt) [inline dev/io.openliberty.org.testcontainers/.gitignore]: I think this change is wrong. The testcontainers cache is auto-updated by the build and is supposed to be checked in, similar to the dependabot pom.

**Review by Azquelt — CHANGES_REQUESTED** [review]
I think the testcontainers cache change is wrong and I also raised one other question.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10/server.xml (hunk 1 above)
Comment (tevans78) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10/server.xml:7]: The test uses restfulWS-3.1, which is pulled in automatically by mpTelemetry-1.0. It doesn't use mpRestClient-3.0.

**Review by Azquelt — APPROVED** [review]

File: dev/io.openliberty.org.testcontainers/.gitignore (hunk 2 above)
Comment (tevans78) [reply dev/io.openliberty.org.testcontainers/.gitignore]: removed these changes

**Review by fmhwong — APPROVED** [review]
LGTM

**Comment by tevans78** [comment]
#libby

**Pull request merged**